Loop/Managers/LoopDataManager.swift

```diff
@@ -94,11 +101,80 @@ class LoopDataManager {
             }
         } catch let error {
             lastLoopError = error
+            self.uploadLoopStatus(nil)
         }
 
         notify()
     }
 
+    private var lastTempBasalUploaded: DoseEntry?
+
+    func uploadLoopStatus(recommendation: TempBasalRecommendation?) {
+
+        let statusTime = NSDate()
+
+        let glucose = deviceDataManager.glucoseStore?.latestGlucose
+
+        let iob: IOBStatus?
+
+        if let iobValues = insulinOnBoardValues, iobValue = iobValues.first {
+            iob = IOBStatus( timestamp: iobValue.startDate, iob: iobValue.value)
+        } else {
+            iob = nil
+        }
+
+       let eventualBG = self.predictedGlucose?.last?.quantity.doubleValueForUnit(HKUnit.milligramsPerDeciliterUnit())
+
+        let loopSuggested: LoopSuggested?
+
+        let glucoseVal = glucose?.quantity.doubleValueForUnit(HKUnit.milligramsPerDeciliterUnit())
+
+        let predBGs: [Double]?
+        if let predicted = predictedGlucose {
+            predBGs = predicted[0...10].map({ (value) -> Double in
```

Comment:
Yep, going to remove that, and also upload the carb and insulin only lines as well

Sent from my iPhone

> On Aug 9, 2016, at 12:47 AM, Mark Wilson <notifications@github.com> wrote:
> 
> In Loop/Managers/LoopDataManager.swift:
> 
> > +
> > +        if let iobValues = insulinOnBoardValues, iobValue = iobValues.first {
> > +            iob = IOBStatus( timestamp: iobValue.startDate, iob: iobValue.value)
> > +        } else {
> > +            iob = nil
> > +        }
> > +
> > +       let eventualBG = self.predictedGlucose?.last?.quantity.doubleValueForUnit(HKUnit.milligramsPerDeciliterUnit())
> > +
> > +        let loopSuggested: LoopSuggested?
> > +
> > +        let glucoseVal = glucose?.quantity.doubleValueForUnit(HKUnit.milligramsPerDeciliterUnit())
> > +
> > +        let predBGs: [Double]?
> > +        if let predicted = predictedGlucose {
> > +            predBGs = predicted[0...10].map({ (value) -> Double in
> Why not include all of them?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>